lint/rule_enabled_test.go

```diff
@@ -87,26 +104,39 @@ func TestRuleIsEnabledFirstMessage(t *testing.T) {
 	// Create a no-op rule, which we can check enabled status on.
 	rule := &FileRule{
 		Name: RuleName("test"),
-		LintFile: func(fd *desc.FileDescriptor) []Problem {
+		LintFile: func(fd protoreflect.FileDescriptor) []Problem {
 			return []Problem{}
 		},
 	}
 
 	// Build a proto and check that ruleIsEnabled does the right thing.
-	f, err := builder.NewFile("test.proto").AddMessage(
-		builder.NewMessage("FirstMessage").SetComments(builder.Comments{
-			LeadingComment: "api-linter: test=disabled",
-		}),
-	).AddMessage(
-		builder.NewMessage("SecondMessage"),
-	).Build()
+	f, err := protodesc.NewFile(&descriptorpb.FileDescriptorProto{
+		Name: proto.String("test.proto"),
+		MessageType: []*descriptorpb.DescriptorProto{
+			{
+				Name: proto.String("FirstMessage"),
+			},
+			{
+				Name: proto.String("SecondMessage"),
+			},
+		},
+		SourceCodeInfo: &descriptorpb.SourceCodeInfo{
+			Location: []*descriptorpb.SourceCodeInfo_Location{
+				{
+					Path:            []int32{4, 0}, // message_type 0
+					Span:            []int32{1, 1, 1, 1},
+					LeadingComments: proto.String(" api-linter: test=disabled"),
```

Comment:
Is this leading white space necessary?
```suggestion
					LeadingComments: proto.String("api-linter: test=disabled"),
```